DSharpPlus/Entities/Guild/DiscordMember.cs

```diff
@@ -223,6 +223,19 @@ public override string Discriminator
         }
 
         /// <summary>
+        /// Gets the member's banner hash.
+        /// </summary>
+        [JsonIgnore]
+        public override string BannerHash
```

Comment:
Also add `BannerUrl`

Reply:
but...why

Reply:
I get it's for consistency but

Reply:
You can't access the user object from `DiscordMember` so it should reflect everything